airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -2109,3 +2128,265 @@ def execute(self, context: Any):
     def on_kill(self):
         if self.job_id and self.cancel_on_kill:
             self.hook.cancel_job(job_id=self.job_id, project_id=self.project_id, location=self.location)
+
+
+class GreatExpectationsBigQueryOperator(BaseOperator):
+    """
+         Use Great Expectations to validate data expectations against a BigQuery table or the result of a SQL query.
```

Comment:
```suggestion
    """
    Use Great Expectations to validate data expectations against a BigQuery table or the result of a SQL query.
```
We should avoid redundant indentation